Makefile

```diff
@@ -6,6 +6,7 @@
 
 BIN_NAME :=krakend
 OS := $(shell uname | tr '[:upper:]' '[:lower:]')
+MODULE := github.com/krakendio/krakend-ce/v2
```

Comment:
We might want to rename `github.com/krakendio/krakend-ce/v2` to `github.com/krakend/krakend-ce/v2`, to match the GitHub handle we have now.

Reply:
@alombarte it has to match the module defined here: https://github.com/krakend/krakend-ce/blob/master/go.mod#L1

Reply:
Hi @LaurentGoderre, apologies. I intended to make an afterthought comment to @kpacha so we can rename all Go packages of KrakenD to reflect the right namespace (this would wait of course until we do another major release and affects all repositories).

Reply:
@alombarte sounds good! I was making sure to document where it came from for others who reads this thread